docset/winserver2025-ps/smbshare/Get-SmbShare.md

```diff
@@ -381,8 +487,9 @@ Accept wildcard characters: False
 ```
 
 ### -SmbInstance
-Specifies the input to this cmdlet.
-You can use this parameter, or you can pipe the input to this cmdlet.
+
+Specifies the input to this cmdlet. You can use this parameter, or you can pipe the input to this
```

Comment:
Specifies the SMB share instance type. The default instance is used for operations like a client opening a file on a share on the server. The CSV instance is used by Windows Failover Clusters. The SBL instance is used by Storage Spaces Direct. The SR instance is used by Storage Replica.